perun-core/src/main/java/cz/metacentrum/perun/core/entry/MembersManagerEntry.java

```diff
@@ -1085,6 +1085,16 @@ public RichMember sponsorMember(PerunSession session, Vo vo, User sponsored, Use
 	}
 
 	@Override
+	public List<RichMember> getSponsoredMembers(PerunSession sess, Vo vo) throws InternalErrorException, PrivilegeException {
+		Utils.checkPerunSession(sess);
+		Utils.notNull(vo, "vo");
+		if(!(AuthzResolver.isAuthorized(sess, Role.REGISTRAR)||AuthzResolver.isAuthorized(sess, Role.VOADMIN, vo))) {
+			throw new PrivilegeException(sess, "getSponsoredMembers must be called by REGISTRAR or VOADMIN");
```

Comment:
Please shorten the exception message to method name: ``getSponsoredMembers``, since this text is wrapped in sentence by exception constructor and passing own sentence would result in nonsense text. Thank you.